update table 2 with updates from #1577

jharenza opened the pull request:
<!--Hi there, thanks for your contribution! Please take a moment to fill out this template to facilitate the review of your pull request.-->

### Purpose

This PR updates Table 2 with the edits in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1577

jharenza commented:
Also closes #297

jashapiro commented on content/03.results.md:
```diff
@@ -227,21 +227,21 @@ Interestingly, none of the hypermutant samples showed evidence of signature 3 (p
-| PT_VTM2STE3               | BS_02YBZSBY               | 7316-2189 | Progressive       | Solid Tissue      | Unknown                                          | Lynch Syndrome                      | None detected               | 274.5 | HGG, H3 wildtype, TP53 activated   |
+| Kids First Participant ID | Kids First Biospecimen ID | CBTN ID   | Phase of therapy  | Composition       | Therapy post-biopsy                              | Cancer predisposition(s)       | Pathogenic germline variant             | TMB                | OpenPBTA molecular subtype         |
+|---------------------------|---------------------------|-----------|-------------------|-------------------|--------------------------------------------------|--------------------------------|-----------------------------------------|--------------------|------------------------------------|
+| PT_0SPKM4S8               | BS_VW4XN9Y7               | 7316-2640 | Initial CNS Tumor | Solid Tissue      | Temozolomide, CCNU, Radiation                    | None documented                | PMS2 p.S46I (LP)                        | 187.38765454966892 | HGG, H3 wildtype, TP53 activated   |
```
Please fix significant figures for TMB.

jharenza replied:
Ah, will do- I should have done this in the actual table.

jharenza replied:
I think to avoid all of the manual changes every time this table gets updated, I'll  open a new PR for this in the analysis repo this afternoon and tag you for review so that we aren't doing double work there and here. Would be good to just paste the table every time.

jashapiro commented on content/03.results.md:
```diff
@@ -227,21 +227,21 @@ Interestingly, none of the hypermutant samples showed evidence of signature 3 (p
+|---------------------------|---------------------------|-----------|-------------------|-------------------|--------------------------------------------------|--------------------------------|-----------------------------------------|--------------------|------------------------------------|
+| PT_0SPKM4S8               | BS_VW4XN9Y7               | 7316-2640 | Initial CNS Tumor | Solid Tissue      | Temozolomide, CCNU, Radiation                    | None documented                | PMS2 p.S46I (LP)                        | 187.38765454966892 | HGG, H3 wildtype, TP53 activated   |
+| PT_3CHB9PK5               | BS_20TBZG09               | 7316-515  | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Bevacizumab, Radiation | Other inherited conditions NOS | MSH6 NM_000179.2 c.3439-2A>G splice (P) | 307.0492167109247  | HGG, H3 wildtype, TP53 loss        |
+| PT_3CHB9PK5               | BS_8AY2GM4G               | 7316-2085 | Progressive       | Solid Tissue      | Unknown                                          | Lynch syndrome                 | MSH6 NM_000179.2 c.3439-2A>G splice (P) | 321.6079467820181  | HGG, H3 wildtype, TP53 loss        |
```
Looking at this long variant description, I'm wondering if we might want to include ClinVar accessions IDs? For this example, it would be `VCV000089391` Alternatively, we could leave all the other the same and `MSH6 c.3439-2A>G (P)` should be sufficient for identification here (it yields a unique search result in ClinVar)

jharenza replied:
I think this may work, but let me confer with Sharon who requested that we put the transcript ID in when showing cDNA changes. It looks like the current record is VCV000089391.27 and lists various transcript IDs, so may be ok.

jashapiro replied:
I don't think the portion after the decimal is required, as that will be updated as information is added to the DB, but will always refer to the same mutation.

jharenza replied:
Hmm ok, yeah I guess I would need the actual clinvar annotation to go with the version. Would it be ok if I just did a line break if the issue is that it's too long?

jashapiro replied:
A line break would be fine. I'm probably more concerned about consistency and easy searching.

jashapiro reviewed (commented):
All of the figure updates here are fine, but I had a couple of suggestions for presentation in table 2.

jharenza commented:
Will update this once https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1597 is approved.

jharenza commented:
@jashapiro - I updated the table, commit number, and added some text around the germline results, so ready for re-review. There may be a need for a dictionary add - I will keep an eye out.

jashapiro commented on content/03.results.md:
```diff
@@ -215,7 +215,10 @@ These findings support previous reports of a more aggressive phenotype in tumors
 Five of six tumors were diagnosed as HGGs and one was a brain metastasis of a MYCN non-amplified neuroblastoma tumor.	
 Signature 11, which has been suggested to result from exposure to temozolomide, was indeed present in tumors with previous exposure to the drug (**Table {@tbl:Table2}**).
-We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with Lynch syndrome (**Table {@tbl:Table2}**), a genetic predisposition syndrome caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.3390/genes5030497].
+We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with either Constitutional mismatch repair deficiency (CMMRD) or Lynch Syndrome (**Table {@tbl:Table2}**), genetic predisposition syndromes caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.1136/jmedgenet-2020-107627].
```
Capitalization.
```suggestion
We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with either constitutional mismatch repair deficiency (CMMRD) or Lynch syndrome (**Table {@tbl:Table2}**), genetic predisposition syndromes caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.1136/jmedgenet-2020-107627].
```

jharenza replied:
Ha, I went back and forth. Lynch Syndrome can be designated LS, and in the table it was both caps, so that is why I had them as caps.

jashapiro commented on content/03.results.md:
```diff
@@ -215,7 +215,10 @@ These findings support previous reports of a more aggressive phenotype in tumors
+We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with either Constitutional mismatch repair deficiency (CMMRD) or Lynch Syndrome (**Table {@tbl:Table2}**), genetic predisposition syndromes caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.1136/jmedgenet-2020-107627].
+Three of these patients harbored pathogenic germline variants in one of the aforementioned genes. 
+While we did not find a _known_ pathogenic variant in the germline of PT_VTM2STE3, this patient had a self-reported _PMS2_ variant noted in their pathology report and we did find 19 intronic variants of unknown significance (VUS) in _PMS2_.
+This is not surprising since an estimated 49% of germline _PMS2_ variants in patients with CMMRD and/or Lynch Syndrome are VUS [@doi:10.1136/jmedgenet-2020-107627].
```
```suggestion
This is not surprising since an estimated 49% of germline _PMS2_ variants in patients with CMMRD and/or Lynch syndrome are VUS [@doi:10.1136/jmedgenet-2020-107627].
```

jashapiro reviewed (approved):
This looks fine to me, with a couple small capitalization updates
